### PR TITLE
BZ-1743299: Clarifying how to find what service accounts an SCC is ap…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…plied to

The PR for the 4.x version of !17686.

### CODE REVIEW

File: modules/security-context-constraints-command-reference.adoc [hunk 1]
```diff
@@ -84,6 +85,8 @@ Settings:
   Supplemental Groups Strategy: RunAsAny
     Ranges:				<none>
 ----
+<1> Lists which users and service accounts that the SCC is applied to.
```
Comment (bobfuru) [inline modules/security-context-constraints-command-reference.adoc]: In this case (and the one below), "that" sounds odd to me.
s/Lists which users and service accounts that the SCC is applied to./Lists which users and which service accounts the SCC is applied to.

File: modules/security-context-constraints-command-reference.adoc [hunk 2]
```diff
@@ -84,6 +85,8 @@ Settings:
...
+<2> Lists which groups that the SCC is applied to.
```
Comment (bobfuru) [inline modules/security-context-constraints-command-reference.adoc]: similar to above
s/groups that the/groups the/

**Review by bobfuru — COMMENTED** [review]
A couple small suggestions (take or leave).

**Comment by bergerhoffer** [comment]
@barleyer Can you please review this? This to update the 4.1-4.3 docs.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1743299

**Pull request merged**